packages/components/src/toolbar/toolbar/index.tsx

```diff
@@ -19,23 +19,30 @@ import type { ToolbarProps } from './types';
 import type { WordPressComponentProps } from '../../context';
 import { ContextSystemProvider } from '../../context';
 
-const CONTEXT_SYSTEM_VALUE = {
-	DropdownMenu: {
-		variant: 'toolbar',
-	},
-	Dropdown: {
-		variant: 'toolbar',
-	},
-};
-
 function UnforwardedToolbar(
 	{
 		className,
 		label,
+		variant,
 		...props
 	}: WordPressComponentProps< ToolbarProps, 'div', false >,
 	ref: ForwardedRef< any >
 ) {
+	const CONTEXT_SYSTEM_VALUE = useMemo( () => {
+		if ( variant !== undefined ) {
+			return {};
```

Comment:
Probably returning `undefined` is better than an empty object in this case

Reply:
That was my initial thought, but that resulted in a type error: 

>Type '{ DropdownMenu: { variant: string; }; Dropdown: { variant: string; }; } | undefined' is not assignable to type 'Record<string, any>'.
>Type 'undefined' is not assignable to type 'Record<string, any>'.

Any ideas?

Reply:
Ok, it looks like the `ContextSystemProvider` component doesn't accept an undefined `value`. We can keep things as they are :)